<?xml version="1.0" encoding="UTF-8"?>
<!--
DFF - An Open Source Digital Forensics Framework
Copyright (C) 2009-2013 ArxSys

This program is free software, distributed under the terms of
the GNU General Public License Version 2. See the LICENSE file
at the top of the source tree.

See http://www.digital-forensic.org for more information about this
project. Please do not directly contact any of the maintainers of
DFF for assistance; the project provides a web site, mailing lists
and IRC channels for your use.

Author(s):
Christophe Malinge cma@digital-forensic.org
-->
<ui version="4.0">
 <class>IdeWizard</class>
 <widget class="QWizard" name="IdeWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Integrated Development Environment Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui.qrc">
    <normaloff>:/ide.png</normaloff>:/ide.png</iconset>
  </property>
  <widget class="QWizardPage" name="introPage">
   <property name="title">
    <string>General Information</string>
   </property>
   <property name="subTitle">
    <string>Fill all script information such as name, type and saving location.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="categoryLabel">
      <property name="text">
       <string>Category:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="category">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="workLocationLabel">
      <property name="text">
       <string>Work location:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="workLocationWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="path"/>
       </item>
       <item>
        <widget class="QPushButton" name="brwButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QWidget" name="typeWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="type_script">
         <property name="text">
          <string>Script</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="type_graphical">
         <property name="text">
          <string>Graphical</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="type_driver">
         <property name="text">
          <string>Driver</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="descriptionPage">
   <property name="title">
    <string>Description</string>
   </property>
   <property name="subTitle">
    <string>Describe here the goal of the module</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Module's description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="description"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="authorPage">
   <property name="title">
    <string>Author informations</string>
   </property>
   <property name="subTitle">
    <string>Fill all author's informations in order to complete header</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Author's first name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Author's last name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Author's electronic mail:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="auth_fname"/>
      </item>
      <item>
       <widget class="QLineEdit" name="auth_lname"/>
      </item>
      <item>
       <widget class="QLineEdit" name="auth_mail"/>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>auth_fname</zorder>
   <zorder>label_4</zorder>
   <zorder>auth_lname</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>auth_mail</zorder>
  </widget>
 </widget>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
